app/core/api/serializers/user.py

```diff
@@ -1,9 +1,41 @@
 from rest_framework import serializers
 from app.core.models import Person
 
+
 class PersonSerializer(serializers.ModelSerializer):
-    username = serializers.CharField(source='user.username', read_only=True)
+    username = serializers.CharField(source="user.username", read_only=True)
+
+    class Meta:
+        model = Person
+        fields = ["id", "username", "avatar", "full_name", "is_active"]
+
 
+class UserDetailUpdateSerializer(serializers.ModelSerializer):
     class Meta:
```

Comment:
Add read_only_fields = ["created_at", "modified_at", "person_id"] and remove is_loggable and user from the fields list.